scene/gui/tab_bar.cpp

```diff
@@ -1001,8 +1005,12 @@ Variant TabBar::get_drag_data(const Point2 &p_point) {
 		return Variant();
 	}
 
-	int tab_over = get_tab_idx_at_point(p_point);
+	// Allow for TabContainer to override it.
+	if (has_meta("_tab_container")) {
+		return get_parent()->call("_get_drag_data_fw", p_point);
+	}
```

Comment:
I'm open to ideas on how to make this less hacky.

Reply:
Something like `set_drag_forwarding(tab_bar)` in the TabContainer? Not sure if it overrides `get_drag_data()` though.

Reply:
Okay, I used that method, now it's done like this:
```
if (!drag_to_rearrange_enabled) {
	return Control::get_drag_data(p_point); // Allow stuff like TabContainer to override it.
}
```